test/end-to-end/README.md

```diff
@@ -37,30 +37,41 @@ $ npm install                                   # Install end-to-end dependencie
 $ npm run test                                  # Run end-to-end test
 ```
 
-```shell
-$ jest -i -t "#acceptance"                               # Run acceptance case only
-$ jest -i -t "@create-recipe-page"                       # Run create recipe page related case only
-$ jest -i -t "#acceptance.+@create-recipe-page"          # Run acceptance case in create recipe page cases
-$ jest -i -t "@create-recipe-page|@edit-recipe-page"     # Run create recipe and edit recipe pages cases
-```
-
 ### Running end-to-end test in Docker
 
 The end-to-end test docker image is an executable image, which starts container, runs test, and exits.
 
-The docker image depends on a base image, named weld/fedora:25, which needs have been previously built.
-If it is not available it can be built from the welder-deployment repository by running `make weld-f25`.
+The docker image depends on a image, named welder/web-nodejs, which needs have been previously downloaded by running:
+```shell
+sudo docker pull welder/web-nodejs:latest
+```
 
 Build end-to-end test docker image by running:
+```shell
+sudo docker build -t weld/end-to-end:latest .
+```
 
-`sudo docker build -t weld/end-to-end:latest .`
+The end-to-end test needs both bdcs-api-rs and welder-web docker running. Download bdcs-api-rs and welder-web docker image by running:
+```shell
+sudo docker pull welder/bdcs-api-rs:latest
+sudo docker pull welder/web:latest
+```
 
-The end-to-end test needs both bdcs-api-rs and welder-web docker running. Build them from welder-deployment
-repository by running `make build` and `make import-metadata`. Run them by running `sudo docker-compose -p welder up -d`.
+Run bdcs-api-rs and web containers:
+```shell
+sudo docker network inspect welder >/dev/null 2>&1 || sudo docker network create welder
+sudo docker ps --quiet --all --filter 'ancestor=welder/bdcs-api-rs' | sudo xargs --no-run-if-empty docker rm -f
+sudo docker run -d --name api --restart=always -p 4000:4000 -v bdcs-recipes-volume:/bdcs-recipes -v `pwd`:/mddb --network welder --security-opt label=disable welder/bdcs-api-rs:latest
+sudo docker run -d --name web -p 3000:3000 --restart=always --network welder welder/web-with-coverage:latest
+```
 
 Run end-to-end test like this:
 
-`sudo docker run --rm --name welder_end_to_end --network welder_default weld/end-to-end:latest xvfb-run -a -s '-screen 0 1024x768x24' npm run test`
+```shell
+sudo docker run --rm --name welder_end_to_end --network host \
+ -v`pwd`/failed-image:/tmp/failed-image \
+ welder/web-e2e-tests:latest xvfb-run -a -s '-screen 0 1024x768x24' npm run test -- --verbose
+```
 
```

Comment:
Keeping this doc in sync with the Makefile seems like a tedious and error-prone task. Why not simply refer to the Makefile targets?

Reply:
Yeah, good suggestion. Updated already. Thanks.